packages/lisk-transactions/test/1_register_second_passphrase.ts

```diff
@@ -132,10 +132,10 @@ describe('#registerSecondPassphrase transaction', () => {
 			).and.not.be.empty;
 		});
 
-		it.skip('should not have a signSignature property', () => {
-			return expect(registerSecondPassphraseTransaction).not.to.have.property(
-				'signSignature',
-			);
+		it('should have a signSignature property when passphrase is provided', () => {
+			return expect(
+				registerSecondPassphraseTransaction.signSignature,
+			).not.to.be.eql(null);
```

Comment:
probably it's better to expect something, and if we can't then 
`not.to.be.undefined`

Reply:
Done. Resolving.